docs/compose/ChipGroup/README.md

```diff
@@ -1,27 +1,29 @@
 # ChipGroup
 
 [![Maven Central](https://img.shields.io/maven-central/v/net.skyscanner.backpack/backpack-compose)](https://search.maven.org/artifact/net.skyscanner.backpack/backpack-compose)
-[![Class reference](https://img.shields.io/badge/Class%20reference-Android-blue)](https://backpack.github.io/android/backpack-compose/net.skyscanner.backpack.compose.chipgroup)
-[![Source code](https://img.shields.io/badge/Source%20code-GitHub-lightgrey)](https://github.com/Skyscanner/backpack-android/tree/main/backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/chipgroup)
 
-## Single Select Rail
+## Single Select
+[![Class reference](https://img.shields.io/badge/Class%20reference-Android-blue)](https://backpack.github.io/android/backpack-compose/net.skyscanner.backpack.compose.chipgroup.single)
```

Comment:
you can also just link to the parent package here and that'll cover both. at some point in the future we wanna look at how we could automate parts of the readme, so keeping the general structure makes sense. we always have the screenshots at the top for all examples, then code examples